content/posts/ubuntu.md

```diff
@@ -0,0 +1,96 @@
+---
+Categories: [Arrangementer]
+Description: "Introduksjonsarrangement"
+Tags: []
+date: 2015-08-09T17:31:16+02:00
+title: "Få et forsprang på livet som programmerer: Del 1"
+---
+
+Dato: 10.09.2015  
+Sted: Fortress @ Ole Johan Dahls  
+[Arrangement på facebook](https://www.facebook.com/events/550620871758803/)
+
+*Artikkelen under er en fullstendig oppsummering av hva som skal skje på arrangementet.*
```

Comment:
Dette stemmer kanskje ikke foreløpig. Om det blir mye venting er det mye man kan si om Linux/Ubuntu for de som ikke vet så mye.